@types/cheerio.d.ts

```diff
@@ -0,0 +1,324 @@
+// Type definitions for Cheerio v0.22.0
```

Comment:
Override only what you really want, and don't the entire file. I'm pretty sure that's possible, see the following:

```ts
import Cherio from "cheerio"

declare module "cheerio" {
  interface Cheerio {
    toArray...
  }
}
```

Reply:
I would suggest telling which is the reason for this file is here.

Reply:
It complains that Cheerio module can't be augmented =[